.github/CONTRIBUTING.md

```diff
@@ -52,33 +55,8 @@ Do you have software fixes or documentation changes to contribute to the codebas
 
 For more robust information about the above steps, see the [Development Guide](#development-guide) below.
 
-
 ### Software Versioning
-The SDR follows the practice of [Semantic Versioning](https://semver.org/) for software releases, tracking the versioning for Geoblacklight where possible.
-The version for a given release is recorded in the `VERSION` file in the project root.
-
-### Development Guide
```

Comment:
Removing the Development Guide since it's a duplication of what's in the main README.